charts/civil-ccd/values.preview.template.yaml

```diff
@@ -3,7 +3,7 @@ civil-wa:
 
 civil-service:
```

Comment:
I also suggest modifying jenkins_cnp to point to camunda branch with your changes